src/components/new-safe/create/steps/ReviewStep/index.tsx

```diff
@@ -45,17 +45,17 @@ import { useMemo, useState } from 'react'
 export const NetworkFee = ({
   totalFee,
   chain,
-  willRelay,
+  strikethrough,
```

Comment:
`strikethrough` refers to how it looks instead of its function. It should be something like `isWaived` instead.